<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIHomework</class>
 <widget class="QWidget" name="UIHomework">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>183</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HL_Content" stretch="0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="lbl_Picture">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>66</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>66</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>图片</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_LeftSpace">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>66</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>66</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(242,242,242)</string>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="VL_Content_2" native="true">
      <layout class="QVBoxLayout" name="VL_Content">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_TopSpace">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>9</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>9</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(242,242,242)</string>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>11</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_Content">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(242,242,242)</string>
         </property>
         <property name="text">
          <string>作业内容的萨达萨达撒多的撒打算多撒</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_Sign">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(120,120,120);</string>
         </property>
         <property name="text">
          <string>发布作业</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_BottomSpace">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>10</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(242,242,242)</string>
         </property>
         <property name="text">
          <string>11</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbl_RightSpace">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>10</width>
     <height>65</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>65</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>65</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(242,242,242)</string>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
